iidm/iidm-mergingview/src/main/java/com/powsybl/iidm/mergingview/MergingView.java

```diff
@@ -714,17 +714,17 @@ public Line getLine(final String id) {
     // DanglingLines
     @Override
     public Iterable<DanglingLine> getDanglingLines() {
-        return Collections.unmodifiableCollection(index.getDanglingLines());
+        return index.getDanglingLines();
     }
 
     @Override
     public Stream<DanglingLine> getDanglingLineStream() {
-        return index.getDanglingLines().stream();
+        return index.getDanglingLineStream();
     }
 
     @Override
     public int getDanglingLineCount() {
-        return index.getDanglingLines().size();
+        return (int) index.getDanglingLineStream().count();
```

Comment:
Same kind of remark: we could sum the danglingLine count, avec remove 2 times the mergedLine. (1 merged Line = 2 danglingLine).